src/contractTest/java/uk/gov/hmcts/reform/sscs/consumer/PartiesNotifiedPutPactConsumerTest.java

```diff
@@ -0,0 +1,217 @@
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.sscs.model.partiesnotified.PartiesNotified;
+import uk.gov.hmcts.reform.sscs.service.HmcHearingPartiesNotifiedApi;
+import uk.gov.hmcts.reform.sscs.utility.BasePactTest;
+
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.CONSUMER_NAME;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.FIELD_ERRORS;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.FIELD_ID;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.FIELD_MESSAGE;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.FIELD_STATUS;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.IDAM_OAUTH2_TOKEN;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.MSG_400_HEARING;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.MSG_401_HEARING;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.MSG_403_HEARING;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.MSG_404_HEARING;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.PARTIES_NOTIFIED_PATH;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.PROVIDER_NAME;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.SERVICE_AUTHORIZATION_TOKEN;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.VALID_CASE_ID;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.authorisedHeaders;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.generateInvalidPartiesPutRequest;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.generatePartiesPutRequest;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.toJsonString;
+import static uk.gov.hmcts.reform.sscs.ContractTestDataProvider.unauthorisedHeaders;
+
+
+@ExtendWith(PactConsumerTestExt.class)
+@EnableFeignClients(basePackages = {"uk.gov.hmcts.reform.sscs.service"})
+@ActiveProfiles("contract")
+@SpringBootTest
+@PactTestFor(port = "10000")
+@PactFolder("pacts")
+class PartiesNotifiedPutPactConsumerTest extends BasePactTest {
+    public static final String FIELD_STATUS_FORMAT = "%d %s";
+    private static final String VERSION = "version";
+    private static final Long VERSION_NUMBER = 123L;
+    private static final String FORBIDDEN_CASE_ID = "456";
+    private static final String NOT_FOUND_CASE_ID = "789";
+    public static final String QUERY_FORMAT = "%s=%s&%s=%s";
```

Comment:
Why public?

Reply:
I guess it is better made private. Done.